i18n/en_CA/LC_MESSAGES/src/containers/Home.js.po

```diff
@@ -0,0 +1,218 @@
+msgstr ""
+"Type in a person's profile to find them. Search is currently case "
+"sensitive"
+
+#: 92:23 src/containers/Home.js
+msgid "This has no exclamation marks"
+msgstr "Get started with the new Directory!"
+
+#: 20:20 src/containers/Home.js
+msgid "Self-serve contact information"
+msgstr ""
+
+#: 22:17 src/containers/Home.js
+msgid ""
+"Easily update your contact information so that it is always accurate. You"
+" can make changes yourself and they will show up immediately."
+msgstr ""
+
+#: 28:20 src/containers/Home.js
+msgid "Easy to search"
+msgstr ""
+
+#: 30:17 src/containers/Home.js
+msgid "Looking for someone? Quickly find other members and start collaborating."
+msgstr ""
+
+#: 36:20 src/containers/Home.js
+msgid "Clean and simple profile"
+msgstr ""
+
+#: 38:17 src/containers/Home.js
+msgid ""
+"Your profile is yours to complete. Add a profile photo to make it "
+"personal."
+msgstr ""
+
+#: 44:20 src/containers/Home.js
+msgid "Create teams"
+msgstr ""
+
+#: 46:17 src/containers/Home.js
+msgid "Supervisors can create teams and self-manage members."
+msgstr ""
+
+#: 52:20 src/containers/Home.js
+msgid "Organizational chart"
+msgstr ""
+
+#: 54:17 src/containers/Home.js
+msgid "Explore an entire team and find the exact person you need to talk to."
+msgstr ""
+
+#: 60:20 src/containers/Home.js
+msgid "New interface design"
+msgstr ""
+
+#: 62:17 src/containers/Home.js
+msgid ""
+"A bright-blue colour scheme, rounded elements and minimalist look make "
+"the application just a little more fun to use."
+msgstr ""
+
+#: 84:20 src/containers/Home.js
+msgid ""
+"Welcome. You’re one of the first to try out the new Directory application"
+" proof of concept. Click one of the options below to jump right in and "
+"explore, or, if you want to learn more, keep reading."
+msgstr ""
+
+#: 90:21 src/containers/Home.js
+msgid "Key features"
+msgstr ""
+
+#: 94:23 src/containers/Home.js
+msgid "What’s coming next?"
+msgstr ""
+
+#: 96:18 src/containers/Home.js
+msgid ""
+"This is the first version of the Directory but we have big plans for the "
+"future. Below is a list of some features we have planned for development."
+msgstr ""
+
+#: 98:18 src/containers/Home.js
+msgid ""
+"Please note that all these features are subject to change, as future "
+"features and improvements will depend on your feedback."
+msgstr ""
+
+#: 104:23 src/containers/Home.js
+msgid "How to provide your feedback"
+msgstr ""
+
+#: 106:18 src/containers/Home.js
+msgid ""
+"We want your feedback to help us improve Directory and priorities new "
+"features to be developed. Before telling us what you think, play around "
+"and test out the new design and functionality of Directory. Take note of "
+"what you like, don’t like, and suggestions for improvement."
+msgstr ""
+
+#: 108:18 src/containers/Home.js
+msgid ""
+"We will be hosting user testing and feedback sessions where you can share"
+" what you’ve discovered. Join the (x) GCcollab group and keep an eye out "
```

Comment:
More a question than a change request but do we have the group on Collab?Or we keep (x) for now?

Reply:
Group link: https://gccollab.ca/groups/profile/2104442/enresearch-participants-open-accessible-digital-workspacefrparticipants-de-recherche-u2013-espace-de-travail-numu00e9rique-ouvert-et-accessible

Reply:
Link to add is:https://gccollab.ca/groups/profile/2104442/enresearch-participants-open-accessible-digital-workspacefrparticipants-de-recherche-u2013-espace-de-travail-numu00e9rique-ouvert-et-accessible